server/routes/api/index.js

```diff
@@ -1,8 +1,11 @@
 import express from 'express';
 import userRoutes from './users';
+import Token from '../../helpers/Token';
+import UserController from '../../controllers/users';
 
 const apiRoutes = express.Router();
 
-apiRoutes.use('/', userRoutes);
+apiRoutes.use('/auth', userRoutes);
+apiRoutes.use('/users/:followid/follow', Token.verifyToken, UserController.follow);
```

Comment:
shouldn't this be:
`apiRoutes.post` ?

However, 

- Create a profiles routes file

change this to:
`apiRoutes.use('/profile', Token.verifyToken, <user routes router>)`

and define the follow route there
`profileRoutes.post('/:username/follow', UserController.follow);`

Reply:
Alright!